common/environment/build-style/cargo.sh

```diff
@@ -1,5 +1,9 @@
 hostmakedepends+=" cargo"
 
+if [ "$pkgname" != "cargo-auditable" ] && [ "$pkgname" != "cargo-auditable-bootstrap" ]; then
```

Comment:
```
❯ ./xbps-src -Q -s -j 16 pkg cargo-auditable          
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://repo-default.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://repo-default.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://repo-default.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://repo-default.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://repo-default.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> cargo-auditable-0.5.2_1: removing autodeps, please wait...
=> cargo-auditable-0.5.2_1: building [cargo] [rust] for x86_64...
   [host] cargo-auditable-bootstrap-0.5.2_1: found (/host/binpkgs/auditable-rust-binaries)
   [host] cargo-1.63.0_1: found (https://repo-default.voidlinux.org/current)
   [host] cargo-auditable-0.5.2_1: found (/host/binpkgs/auditable-rust-binaries)
=> cargo-auditable-0.5.2_1: installing host dependencies: cargo-auditable-bootstrap-0.5.2_1 cargo-1.63.0_1 cargo-auditable-0.5.2_1 ...
=> cargo-auditable-0.5.2_1: removing autodeps, please wait...
=> ERROR: cargo-auditable-0.5.2_1: failed to install host dependencies! (error 11)
CONFLICT: cargo-auditable-0.5.2_1 with cargo-auditable-bootstrap-0.5.2_1 in transaction (matched by cargo-auditable-bootstrap)
Transaction aborted due to conflicting packages.
=> ERROR: Please see above for the real error, exiting...
```

Reply:
doesn't look like the bashism really does what we want though

Reply:
without the slashes it works as expected.

Reply:
Hmm, I don't like this `~=`. It's very unlikely there will be a package called cargo-auditable-something that would be wrongly caught by this, but it still feels like the wrong approach. Is there a reason why you prefer `~=` instead of just listing both packages @classabbyamp?

Reply:
idk I thought it looked better

Reply:
I think that if there are 2 names we want to match, we should match just those, not more.

Reply:
I guess if we wanted to keep the regex, `^cargo-auditable(-bootstrap)?$` is the way to go

Reply:
Adjusted the regex accordingly.